packages/v4/patternfly-docs/content/design-guidelines/components/tooltip/tooltip.md

```diff
@@ -20,17 +20,17 @@ Use tooltips to:
 
 <img src="./img/tooltip-disabled-button.png" alt= 'disabled edit button with a tooltip about access' width='1500'/>
 
-## Tooltip delay on hover
+### Tooltip delay on hover
 By default, tooltips have a 300ms entry delay when hovering over an object. Although this is the recommended delay time, you may customize it via prop. For example, if you have a tooltip on an icon, and you would like for it to appear immediately on hover, you may set the delay to 0 ms.
 
-## Tooltips versus popovers
+### Tooltips vs. popovers
```

Comment:
Would just edit this header to say "When to use tooltips vs. popovers" to match the headers in other guidelines